src/main/java/lotto/LottoGameApplication.java

```diff
@@ -19,11 +19,12 @@ public static void main(String[] args) {
     ResultView.printPurchase(purchase);
 
     String winningLottoNumbers = InputView.inputWinningLottoNumbers();
-    LottoTicket winning = LottoTicket.from(winningLottoNumbers);
+    int bonusNumber = InputView.inputBonusNumber();
+    WinningTicket winning = new WinningTicket(winningLottoNumbers, bonusNumber);
 
     LottoGame game = new LottoGame(winning);
     GameResult gameResult = game.check(purchase);
```

Comment:
```suggestion
LottoTickets tickets = LottoMachine.generateTickets(purchase); // purchase에서 구매할 ticket 수를 구해 LottoTicket을 생성해 반환
GameResult gameResult = winning.match(tickets);
또는 
GameResult gameResult = tickets.match(winning);
```
위와 같이 접근하면 어떨까?
현재와 같이 구현했을 때와 위와 같이 구현했을 때 어떤 차이점이 있는지 고민해 볼 것을 추천한다.
테스트 측면에서도 어떻게 구현하는 것이 나은지도 고민해 본다.

Reply:
이 피드백 다시 한번 고민해 보면 어떨까?

Reply:
- LottoGame.check(purchase) 내부에서 티켓 생성이 일어나면, 랜덤 생성 로직이 포함되어 테스트가 어려움
  - 중간 계층이 많아 단위 테스트 작성 시 모킹이 필요해짐
2가지가 있을 것 같습니다